fonts.gradle

```diff
@@ -2,57 +2,55 @@
  * Task to copy icon font files
  */
 
-def config = project.hasProperty("vectoricons") ? project.vectoricons : [];
 
-def iconFontsDir = config.iconFontsDir ?: "../../node_modules/react-native-vector-icons/Fonts";
-def iconFontNames = config.iconFontNames ?: [ "*.ttf" ];
+afterEvaluate {
+    def config = project.hasProperty("vectoricons") ? project.vectoricons : [];
+    def iconFontsDir = config.iconFontsDir ?: "../../node_modules/react-native-vector-icons/Fonts";
+    def iconFontNames = config.iconFontNames ?: [ "*.ttf" ];
+
 
-gradle.projectsEvaluated {
     android.applicationVariants.all { def variant ->
         def targetName = variant.name.capitalize()
         def targetPath = variant.dirName
 
         // Create task for copying fonts
-        def currentFontTask = tasks.create(
-                name: "copy${targetName}IconFonts",
-                type: Copy) {
-            into("${buildDir}/intermediates")
+        def currentFontCopyTask = tasks.create(
+            name: "copy${targetName}ReactNativeVectorIconFonts",
+            type: Copy) {
+            group = "react"
+            description = "copy fonts into ${targetName}."
 
-            iconFontNames.each { fontName ->
+            into("$buildDir/intermediates")
+            // println("[react-native-vector-icons] Copying font resources in release build")
 
-              from(iconFontsDir) {
-                include(fontName)
-                into("assets/${targetPath}/fonts/")
-              }
-
-              // Workaround for Android Gradle Plugin 3.2+ new asset directory
-              from(iconFontsDir) {
-                include(fontName)
-                into("merged_assets/${variant.name}/merge${targetName}Assets/out/fonts/")
-              }
-
-              // Workaround for Android Gradle Plugin 3.4+ new asset directory
-              from(iconFontsDir) {
-                include(fontName)
-                into("merged_assets/${variant.name}/out/fonts/")
-              }
+            iconFontNames.each { fontName ->
+                //  println("  ...copying ${fontName}")
```

Comment:
Same

Reply:
Oh of course! Sorry :-)

Reply:
@oblador re-pushed with no commented-out "quality assurance" statements ;-), and even more beautiful spacing